### PR TITLE
polish(lxl): polish mujoco(hopper,walker2d) config

### PR DESCRIPTION
**Pull request by lixl-st**
## Check List

- [ ] merge the latest version source branch/repo, and resolve all the conflicts
- [ ] pass style check
- [ ] pass all the tests

### CODE REVIEW

**Comment by puyuan1996** [comment]
please modify PR's name according to our guideline，something like this:
polish(lxl): polish mujoco(hopper,walker2d) config

**Comment by uuid0000** [comment]
I propose for m1 (arm64) to be added to `do_platform_test`
so many failed wheels with default set-ups......

File: dizoo/mujoco/config/hopper_trex_onppo_config.py [hunk 1]
```diff
@@ -57,16 +63,21 @@
...
     env_manager=dict(type='subprocess'),
     policy=dict(type='ppo', ),
 )
-hopper_trex_ppo_create_default_config = EasyDict(hopper_trex_ppo_create_default_config)
-create_config = hopper_trex_ppo_create_default_config
+hopper_trex_onppo_create_config = EasyDict(hopper_trex_onppo_create_config)
+create_config = hopper_trex_onppo_create_config
+
+
+if __name__ == "__main__":
```
Comment (puyuan1996) [inline dizoo/mujoco/config/hopper_trex_onppo_config.py:81]: add line # or you can enter `ding -m serial_trex_onpolicy -c hopper_trex_onppo_config.py -s 0`

File: dizoo/mujoco/config/hopper_trex_sac_config.py [hunk 2]
```diff
@@ -78,5 +84,10 @@
     ),
     replay_buffer=dict(type='naive', ),
 )
-hopper_trex_sac_default_create_config = EasyDict(hopper_trex_sac_default_create_config)
-create_config = hopper_trex_sac_default_create_config
+hopper_trex_sac_create_config = EasyDict(hopper_trex_sac_create_config)
+create_config = hopper_trex_sac_create_config
+
+
+if __name__ == "__main__":
```
Comment (puyuan1996) [inline dizoo/mujoco/config/hopper_trex_sac_config.py:91]: add line # or you can enter `ding -m serial_trex -c hopper_trex_sac_config.py -s 0`

File: dizoo/mujoco/config/walker2d_trex_onppo_config.py [hunk 3]
```diff
@@ -57,16 +63,21 @@
...
     env_manager=dict(type='subprocess'),
     policy=dict(type='ppo', ),
 )
-walker_trex_ppo_create_default_config = EasyDict(walker_trex_ppo_create_default_config)
-create_config = walker_trex_ppo_create_default_config
+walker2d_trex_onppo_create_config = EasyDict(walker2d_trex_onppo_create_config)
+create_config = walker2d_trex_onppo_create_config
+
+
+if __name__ == "__main__":
```
Comment (puyuan1996) [inline dizoo/mujoco/config/walker2d_trex_onppo_config.py:81]: add line # or you can enter `ding -m serial_trex_onpolicy -c walker2d_trex_onppo_config.py -s 0`

File: dizoo/mujoco/config/walker2d_trex_sac_config.py [hunk 4]
```diff
@@ -78,5 +84,10 @@
     ),
     replay_buffer=dict(type='naive', ),
 )
-walker2d_trex_sac_default_create_config = EasyDict(walker2d_trex_sac_default_create_config)
-create_config = walker2d_trex_sac_default_create_config
+walker2d_trex_sac_create_config = EasyDict(walker2d_trex_sac_create_config)
+create_config = walker2d_trex_sac_create_config
+
+
+if __name__ == "__main__":
```
Comment (puyuan1996) [inline dizoo/mujoco/config/walker2d_trex_sac_config.py:91]: add line # or you can enter `ding -m serial_trex -c walker2d_trex_sac_config.py -s 0`

File: dizoo/mujoco/config/hopper_trex_onppo_config.py [hunk 5]
```diff
@@ -79,5 +79,6 @@
 
 
 if __name__ == "__main__":
+    # or you can enter ding -m serial_trex_onpolicy -c hopper_trex_onppo_config.py -s 0
```
Comment (puyuan1996) [inline dizoo/mujoco/config/hopper_trex_onppo_config.py:82]: 逆强化学习的config, 请先merge最新的main分支，然后if __name__ == "__main__":下面的内容需要按照云鹏给的相应示例修改，比如https://github.com/opendilab/DI-engine/blob/main/dizoo/mujoco/config/halfcheetah_trex_onppo_config.py 现在这样写运行应该是报错的吧？

File: dizoo/mujoco/config/walker2d_ppo_config.py [hunk 6]
```diff
@@ -41,16 +41,22 @@
...
     env_manager=dict(type='subprocess'),
     policy=dict(type='ppo', ),
 )
-walker_ppo_create_default_config = EasyDict(walker_ppo_create_default_config)
-create_config = walker_ppo_create_default_config
+walker2d_ppo_create_config = EasyDict(walker2d_ppo_create_config)
+create_config = walker2d_ppo_create_config
+
+
+if __name__ == "__main__":
```
Comment (puyuan1996) [inline dizoo/mujoco/config/walker2d_ppo_config.py:59]: 这个文件和上面的重合了，应该直接删除的。还有在需要在DI-engine目录下执行 bash format.sh ding dizoo，进行yapf格式化。

File: dizoo/mujoco/config/hopper_sac_config.py [hunk 7]
```diff
@@ -46,20 +46,26 @@
...
-hopper_sac_default_create_config = EasyDict(hopper_sac_default_create_config)
-create_config = hopper_sac_default_create_config
+hopper_sac_create_config = EasyDict(hopper_sac_create_config)
+create_config = hopper_sac_create_config
+
+
+if __name__ == "__main__":
+    # or you can enter `ding -m serial -c hopper_sac_config.py -s 0`
+    from ding.entry import serial_pipeline
+    serial_pipeline([main_config, create_config], seed=0)
```
Comment (zjowowen) [inline dizoo/mujoco/config/hopper_sac_config.py:71]: A newline at the end of file is recommended.

Comment (zjowowen) [inline dizoo/mujoco/config/hopper_sac_config.py:71]: A newline at the end of file is recommended.

File: dizoo/mujoco/config/walker2d_td3_config.py [hunk 8]
```diff
@@ -47,20 +48,26 @@
...
-walker2d_td3_default_create_config = EasyDict(walker2d_td3_default_create_config)
-create_config = walker2d_td3_default_create_config
+walker2d_td3_create_config = EasyDict(walker2d_td3_create_config)
+create_config = walker2d_td3_create_config
+
+
+if __name__ == "__main__":
+    # or you can enter `ding -m serial -c walker2d_td3_config.py -s 0`
+    from ding.entry import serial_pipeline
+    serial_pipeline([main_config, create_config], seed=0)
```
Comment (zjowowen) [inline dizoo/mujoco/config/walker2d_td3_config.py:73]: A newline is recommended at the end of file.

Comment (zjowowen) [inline dizoo/mujoco/config/walker2d_td3_config.py:73]: A newline is recommended at the end of file.

File: dizoo/mujoco/config/walker2d_trex_onppo_config.py [hunk 9]
```diff
@@ -0,0 +1,96 @@
+from easydict import EasyDict
+
+walker2d_trex_onppo_config = dict(
+    exp_name='walker2d_trex_onppo_seed0',
+    env=dict(
+        manager=dict(shared_memory=True, reset_inplace=True),
```
Comment (zjowowen) [inline dizoo/mujoco/config/walker2d_trex_onppo_config.py:6]: if both shared_memory=True, reset_inplace=True, this line could be omitted by default.

Comment (zjowowen) [inline dizoo/mujoco/config/walker2d_trex_onppo_config.py:6]: if both shared_memory=True, reset_inplace=True, this line could be omitted by default.

File: dizoo/mujoco/config/walker2d_trex_sac_config.py [hunk 10]
```diff
@@ -0,0 +1,105 @@
+from easydict import EasyDict
+
+walker2d_trex_sac_config = dict(
+    exp_name='walker2d_trex_sac_seed0',
+    env=dict(
+        manager=dict(shared_memory=True, reset_inplace=True),
```
Comment (zjowowen) [inline dizoo/mujoco/config/walker2d_trex_sac_config.py:6]: if both shared_memory=True, reset_inplace=True, this line could be omitted by default.

Comment (zjowowen) [inline dizoo/mujoco/config/walker2d_trex_sac_config.py:6]: if both shared_memory=True, reset_inplace=True, this line could be omitted by default.

File: dizoo/mujoco/config/hopper_gcl_config.py [hunk 11]
```diff
@@ -50,20 +48,22 @@
         eval=dict(evaluator=dict(eval_freq=100, )),
```
Comment (zjowowen) [inline dizoo/mujoco/config/hopper_gcl_config.py:48]: "demonstration_info_path='path'" should be replaced by new model_path format.

Comment (zjowowen) [inline dizoo/mujoco/config/hopper_gcl_config.py:48]: "demonstration_info_path='path'" should be replaced by new model_path format.

**Review by PaParaZz1 — APPROVED** [review]

**Pull request merged**